microsoft-edge/webview2/concepts/webview-features-flags.md

```diff
@@ -114,6 +114,7 @@ The following are some of the flags we've seen used.
 | `msEdgeDesignerDriverFix ` |  Use this flag to disable getting content and automatically showing the Designer feature. This feature is not supported in WebView2. |
 | `msEdgeFluentOverlayScrollbar` | Force-enables Fluent Overlay scrollbars, overriding whatever value `kFluentScrollbar` may hold at the moment. |
 | `msEdgeDevToolsWdpRemoteDebugging` | Enables remote debugging.  See [Remote debugging WebView2 WinUI 2 (UWP) apps](../how-to/remote-debugging.md), [Remotely debug Windows devices](../../devtools-guide-chromium/remote-debugging/windows.md). |
+| `msEdgeWebViewApplyWebResourceRequestedFilterForOOPIFs` | If this flag is enabled, the `AddWebResourceRequestedFilter` method (including overloads) will also apply to out-of-process iframes. <br/>.NET: [AddWebResourceRequestedFilter](/dotnet/api/microsoft.web.webview2.core.corewebview2.addwebresourcerequestedfilter) <br/>WinRT: [AddWebResourceRequestedFilter](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2#addwebresourcerequestedfilter) <br/>Win32: [AddWebResourceRequestedFilterWithRequestSourceKinds](/microsoft-edge/webview2/reference/win32/icorewebview2_22#addwebresourcerequestedfilterwithrequestsourcekinds) |
```

Comment:
alt design:
```suggestion
| `msEdgeWebViewApplyWebResourceRequestedFilterForOOPIFs` | If this flag is enabled, the `AddWebResourceRequestedFilter` method (including overloads) will also apply to out-of-process iframes.  For API Reference links, see [Manage network requests in WebView2](./overview-features-apis.md#manage-network-requests-in-webview2) in _Overview of WebView2 features and APIs_. |
```
https://review.learn.microsoft.com/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-3275#manage-network-requests-in-webview2
fyi @nishitha-burman, @monica-ch 

This alt design is:
* Compact - good for table cells, which are extremely restrictive for layout/Markdown.  solves problem of how to cram a tabset into almost mid-sentence compact form.
* Easy to tag & implement.  
* Lists related APIs; gives context, w/ lead-in in "Overview of APIs".

Reply:
continuing to consider this alt design, but closing Comment w/o now applying this alt design.